packages/actions/src/Concerns/CanOpenModal.php

```diff
@@ -510,7 +511,21 @@ public function getModalDescription(): string | Htmlable | null
 
     public function getModalWidth(): string
     {
-        return $this->evaluate($this->modalWidth) ?? '4xl';
```

Comment:
I don't think this line should change, the matching should be done in the view. Please pull the latest changes from main

Reply:
the return type of the method should be updated too